public/docs/_examples/toh-4/ts/app/hero.service.ts

```diff
@@ -12,8 +12,8 @@ export class HeroService {
   getHeroes() {
     return Promise.resolve(HEROES);
   }
-  // #enddocregion get-heroes
-  // #enddocregion just-get-heroes
+  // #enddocregion get-heroes, just-get-heroes
+  // #enddocregion
```

Comment:
Had a conversation with @Foxandxss in a separate thread, and he is ok with the change made here. FYI.